lambda_bucket.tf

```diff
@@ -1,28 +1,41 @@
+# This bucket is used to store the lambda function that imports findings
```

Comment:
I believe we have been capitalizing this:
```suggestion
# This bucket is used to store the Lambda function that imports findings
```

Reply:
Please see [99e1c8c](https://github.com/cisagov/findings-data-import-terraform/pull/36/commits/99e1c8c73f9c91be8c3efeebed29e18000f2b0bd).